hudi-common/src/main/java/org/apache/hudi/BaseHoodieTableFileIndex.java

```diff
@@ -323,30 +424,13 @@ private void doRefresh() {
         .mapToLong(BaseHoodieTableFileIndex::fileSliceSize)
         .sum();
 
-    // If the partition value contains InternalRow.empty, we query it as a non-partitioned table.
-    queryAsNonePartitionedTable = partitionFiles.keySet().stream().anyMatch(p -> p.values.length == 0);
```

Comment:
Let's actually replace this flag w/ `isPartitionedTable` predicate (sorry, it's my bad that i missed it cleaning it up last time)

Reply:
Do you mean remove the variable at all, and have `isPartitionedTable ` to play its functionality?

Reply:
Correct